site2/docs/concepts-messaging.md

```diff
@@ -396,6 +397,57 @@ producer = client.create_producer(topic='my-topic', batching_type=pulsar.Batchin
 > * You need to specify a key or orderingKey for messages.
 > * You cannot use cumulative acknowledgment with Key_Shared type.
 
+### Subscription mode
+
+#### What is subscription mode
```

Comment:
Here, we should either use `a subscription mode` or `subscription modes`. @Anonymitaet

Reply:
`subscription modes`

can you update it in your PR #12794? thanks

Reply:
Done.